pom.xml

```diff
@@ -190,6 +190,24 @@
 									</systemPropertyVariables>
 								</configuration>
 							</execution>
+							<execution>
```

Comment:
Container startup fails with a TNS error message quite regularly. That causes a lot of the CI builds to fail.

I'm wondering whether we should run DB2 and Oracle tests only in a nightly job.

Reply:
Where do you see these regular failures? In Jenkins I see one failure in the last 14 builds which would gives me hope that just waiting a little longer might fix the problem, especially since it is a TNS failure and not a "your account has been locked".